lib/funky/post.rb

```diff
@@ -0,0 +1,18 @@
+module Funky
+  class Post < GraphRootNode
+    # @return [String] the type of post.
+    def type
+      data[:type]
+    end
+
+    # @return [DateTime] the created time of the post.
+    def created_time
+      DateTime.parse data[:created_time]
+    end
+
+    # @return [Integer] the number of shares of the post.
+    def count_shares
```

Comment:
@kangkyu I think we are using `post_count`, `video_count` etc in our naming. 
So maybe this could be `share_count` ?